datamodules/lucas.py

```diff
@@ -13,13 +13,14 @@
 import os
 
 
+# TODO: check output of collate fn is comparable to the output of my collate fn
```

Comment:
It probably shouldn't be, unless you put the sequence dimension last like this does.

Reply:
Gotcha -- mine is B x T x C, so yeah, they'd be different. Thanks!